src/services/Spotify.js

```diff
@@ -0,0 +1,91 @@
+/* @flow */
+
+const spotifyAPI = require('../lib/spotifyAPI');
+
+const Song = require('../schemas/Song');
+const Artist = require('../schemas/Artist');
+const Album = require('../schemas/Album');
+
+class Spotify {
+
+  // Props
+  api: any
+
+  // Methods
+  constructor() {
+    this.api = spotifyAPI;
```

Comment:
You could inject this dependency from the highest level, so it reads like
```js
constructor(api) {
  this.api = spotifyAPI;
}
```

With this, you make the class easier to test because instead of trying to mock it, you can just inject something else that complies the api.